### PR TITLE
11.0 autostaging base porting 2

### PR DESCRIPTION
**Pull request by Ommo73**

### CODE REVIEW

File: autostaging_base/models.py [hunk 1]
```diff
@@ -118,6 +118,7 @@ def _get_model_list(self):
 
     @api.model
     def _cron_move_cards(self):
+        print('AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA')
```
Comment (yelizariev) [inline autostaging_base/models.py]: :-1:

File: autostaging_project_task/views.xml [hunk 2]
```diff
@@ -49,7 +49,7 @@
 
         <record id="view_project_task_type_tree_autostaging" model="ir.ui.view">
             <field name="name">project.task.type.tree.autostaging</field>
-            <field name="model">project.task.type</field>
+            <field name="model_id" ref="model_project_task_type"/>
```
Comment (yelizariev) [inline autostaging_project_task/views.xml]: <field name="model_id" ref="project.model_project_task_type"/>

?

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: autostaging_base/models.py [hunk 3]
```diff
@@ -44,6 +44,7 @@ class AutostagingCard(models.AbstractModel):
     _field_folder_id = 'define_some_field_folder_id'
     _field_stage_id = 'define_some_field_stage_id'
 
+
```
Comment (KolushovAlexandr) [inline autostaging_base/models.py]: keep the history clean

File: autostaging_base/models.py [hunk 4]
```diff
@@ -70,7 +71,7 @@ def _get_autostaging_date(self):
 
         delta = datetime.timedelta(days=getattr(self, self._field_stage_id).autostaging_idle_timeout)
         return (datetime.datetime.strptime(
-            self.write_date, DEFAULT_SERVER_DATETIME_FORMAT) + delta).strftime(DEFAULT_SERVER_DATETIME_FORMAT)
+            self.write_date,  DEFAULT_SERVER_DATETIME_FORMAT) + delta).strftime(DEFAULT_SERVER_DATETIME_FORMAT)
```
Comment (KolushovAlexandr) [inline autostaging_base/models.py]: keep the history clean

**Review by KolushovAlexandr — CHANGES_REQUESTED** [review]
1 Update Docs 
2 Rebase commits

File: autostaging_base/__manifest__.py [hunk 5]
```diff
@@ -1,12 +1,48 @@
+# Copyright 2018 Ildar Nasyrov <https://it-projects.info/>
```
Comment (KolushovAlexandr) [inline autostaging_base/__manifest__.py]: Wrong, https://github.com/it-projects-llc/misc-addons/commits/12.0/autostaging_base
# Copyright 2015-2017 Ildar Nasyrov <https://it-projects.info/>

add into all required files (*.py, *.xml, *.js) contributors and licenses and don't forget to add in there your name too

File: autostaging_base/README.rst [hunk 6]
```diff
@@ -1,5 +1,50 @@
-Autostaging (technical core)
-============================
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/lgpl
+   :alt: License: LGPL-3
 
+=================
+ Autostaging base
+=================
```
Comment (KolushovAlexandr) [inline autostaging_base/README.rst]: Here and on the one above should be on more '=' after the name ends, look how in the template https://odoo-development.readthedocs.io/en/latest/dev/docs/README.rst.html

File: autostaging_base/__manifest__.py [hunk 7]
```diff
@@ -1,12 +1,48 @@
...
-    'data': [
-        'data.xml',
-    ],
-    "installable": False
+    "name": """Autostaging (technical core)""",
+    "summary": """Change stages of tasks automatically after a specified time""",
+    "category": "Autostaging",
+    # "live_test_url": "http://apps.it-projects.info/shop/product/DEMO-URL?version=11.0",
+    "images": [],
+    "version": "11.0.1.0.1",
```
Comment (KolushovAlexandr) [inline autostaging_base/__manifest__.py]: No need to change version when porting keep 11.0.1.0.0

File: autostaging_base/doc/changelog.rst [hunk 8]
```diff
@@ -0,0 +1,12 @@
+Changelog
+=========
+
+`1.0.1`
```
Comment (KolushovAlexandr) [inline autostaging_base/doc/changelog.rst]: No need to change version when porting keep 11.0.1.0.0

File: autostaging_base/doc/index.rst [hunk 9]
```diff
@@ -0,0 +1,21 @@
+================
+Autostaging base
+================
```
Comment (KolushovAlexandr) [inline autostaging_base/doc/index.rst]: One more '=' sign before the name and after

File: autostaging_base/doc/index.rst [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+
+Installation
+============
+
+* `Install <https://odoo-development.readthedocs.io/en/latest/odoo/usage/install-module.html>`__ this module in a usual way
+
+Configuration
+=============
```
Comment (KolushovAlexandr) [inline autostaging_base/doc/index.rst]: May be removed

File: autostaging_base/doc/index.rst [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+
+Does not require pre-configuration
+
+Usage
+=====
```
Comment (KolushovAlexandr) [inline autostaging_base/doc/index.rst]: This paragraph is related to autostaging_project_task not to this module

File: autostaging_project_task/README.rst [hunk 12]
```diff
@@ -1,4 +1,9 @@
-Autostaging project task
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/lgpl
+   :alt: License: LGPL-3
+
+=========================
```
Comment (KolushovAlexandr) [inline autostaging_project_task/README.rst]: one more '=' is needed

File: autostaging_project_task/__manifest__.py [hunk 13]
```diff
@@ -1,26 +1,50 @@
+# Copyright 2018 Ildar Nasyrov <https://it-projects.info/>
```
Comment (KolushovAlexandr) [inline autostaging_project_task/__manifest__.py]: Update copyrights in all required files

File: autostaging_project_task/doc/index.rst [hunk 14]
```diff
@@ -1,5 +1,5 @@
 ==========================
- Autostaging project task
+Autostaging project task
```
Comment (KolushovAlexandr) [inline autostaging_project_task/doc/index.rst]: Whitespace at the line start is missed

**Review by KolushovAlexandr — CHANGES_REQUESTED** [review]
All remarks are related to document design. Update commits in order to have by only one porting commit for each module

**Pull request closed without merging**